Alignment/OfflineValidation/src/TrackerValidationVariables.cc

```diff
@@ -355,18 +369,3 @@ TrackerValidationVariables::fillTrackQuantities(const edm::Event& event,
   }
 }
 
-void
-TrackerValidationVariables::fillHitQuantities(const edm::Event& event, std::vector<AVHitStruct> & v_avhitout)
-{
-  edm::Handle<std::vector<Trajectory> > trajCollectionHandle;
-  event.getByToken(trajCollectionToken_, trajCollectionHandle);
-  
-  LogDebug("TrackerValidationVariables") << "trajColl->size(): " << trajCollectionHandle->size() ;
-
-  for (std::vector<Trajectory>::const_iterator it = trajCollectionHandle->begin(), itEnd = trajCollectionHandle->end(); 
-       it!=itEnd;
-       ++it) {
-    
-    fillHitQuantities(&(*it), v_avhitout);
-  }
-}
```

Comment:
The comment in the header says it is only used in TrackerMonitorTrack, which now uses the other version. [DXR confirms there are no other callers (as of 7_6_X)](
http://cmslxr.fnal.gov/dxr/CMSSW/search?q=%2Bfunction-ref%3A%22TrackerValidationVariables%3A%3AfillHitQuantities%28const+edm%3A%3AEvent+%26%2C+std%3A%3Avector%3CAVHitStruct%3E+%26%29%22)